manifests/init.pp

```diff
@@ -45,6 +45,7 @@
   Boolean $edit_only                = false,
   Boolean $require_authentication   = false,
   Boolean $require_authorization    = false,
+  Boolean $no_default_ldapauth      = false,
```

Comment:
I'd prefer to use `Boolean default_ldapauth = true` to avoid a double negative.

Reply:
Or maybe `use_default_ldapauth`

Reply:
This is what we agreed with @ekohl on IRC. To use this : 

`if $use_default_ldapauth { merge() } else { simple() }`
with default value `$use_default_ldapauth = true`

@sergiik, if you agree,  can you update your PR accordingly with this collegiate talk ?

Reply:
Agree. Updated as suggested